lib/foreman_remote_execution_core.rb

```diff
@@ -8,9 +8,18 @@ module ForemanRemoteExecutionCore
                     :remote_working_dir    => '/var/tmp',
                     :local_working_dir     => '/var/tmp')
 
+  def self.debug?
+    %w(yes true 1).include? ENV.fetch('REX_DEBUG', '')
```

Comment:
`REX_DEBUG` name is misleading: `REX_SIMULATE` or `REX_FAKE` would more describe what's going on there